patches/core/0026-adb-disable-mDNS.patch

```diff
@@ -0,0 +1,66 @@
+From 47558bcba1688e35cdf761c1656bd0f26a0dd342 Mon Sep 17 00:00:00 2001
+From: =?UTF-8?q?=C4=90o=C3=A0n=20Tr=E1=BA=A7n=20C=C3=B4ng=20Danh?=
+ <congdanhqx@gmail.com>
+Date: Sun, 1 Nov 2020 19:09:53 +0700
+Subject: [PATCH] adb: disable mDNS
+
+---
+ adb/adb.cpp         | 15 ---------------
+ adb/socket_spec.cpp | 18 ------------------
+ 2 files changed, 33 deletions(-)
+
+diff --git a/adb/adb.cpp b/adb/adb.cpp
+index daae0e103..ebc653375 100644
+--- a/adb/adb.cpp
++++ b/adb/adb.cpp
+@@ -1077,21 +1077,6 @@ void adb_set_reject_kill_server(bool value) {
+ }
+ 
+ static bool handle_mdns_request(std::string_view service, int reply_fd) {
+-    if (!android::base::ConsumePrefix(&service, "mdns:")) {
+-        return false;
+-    }
+-
+-    if (service == "check") {
+-        std::string check = mdns_check();
+-        SendOkay(reply_fd, check);
+-        return true;
+-    }
+-    if (service == "services") {
+-        std::string services_list = mdns_list_discovered_services();
+-        SendOkay(reply_fd, services_list);
+-        return true;
+-    }
+-
+     return false;
+ }
+ 
+diff --git a/adb/socket_spec.cpp b/adb/socket_spec.cpp
+index 5cad70d09..f143b6c58 100644
+--- a/adb/socket_spec.cpp
++++ b/adb/socket_spec.cpp
+@@ -196,24 +196,6 @@ bool socket_spec_connect(unique_fd* fd, std::string_view address, int* port, std
+             fd->reset(network_loopback_client(port_value, SOCK_STREAM, error));
+         } else {
+ #if ADB_HOST
```

Comment:
Maybe just replace this with `#if 0` to trigger the `#else`?

Reply:
No, we don't want to jump into `#else` block. The whole block should be removed instead.